### DISCUSSION TITLE
Using Go2RTC and WebRTC through Home Assistant into HomeKit?

### DISCUSSION THREAD

**Discussion by Dropbear67** [blakeblackshear/frigate#6155] [Ask A Question]
I have the frigate plug in in HA and I export my Frigate Cameras through to HomeKit through the Homekit Integration. Will the cameras within Homekit be using WebRTC ? They seem pretty slow to start up in HomeKit and I can't really find a way of being sure if it's using it or not.

**Comment by NickM-27** [comment]
No, homekit does not support WebRTC. It just uses rtsp.
